Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -169,6 +169,7 @@ class TextureCacheBase
 
   TCacheEntry* Load(const u32 stage);
   static void InvalidateAllBindPoints() { valid_bind_points.reset(); }
+  static void SetBindPoint(u8 stage, bool value) { valid_bind_points.set(stage, value); }
```

Comment:
I don't see that bool ever be anything but `false`. Unless you've got something else lined up, it might be nicer to just make it `void InvalidateBindPoint(u8 stage)` instead.